### DISCUSSION TITLE
Is there a more detailed description of the functions of floris? The current documentation is not detailed enough

### DISCUSSION THREAD

**Discussion by tatesnow** [NatLabRockies/floris#242] [Q&A] [answered]
as stated in title

**Comment by Bartdoekemeijer** [comment, marked as answer]
Hi,

Documentation about FLORIS repository can be found [here](https://floris.readthedocs.io/en/main/). Additionally, for more information about the models, we refer you to their related literature. More information about the functions can be find within the function descriptions and in the example files.

Is there anything specific you would like to know more about?

With kind regards,
Bart
